assets/js/util.js

```diff
@@ -39,6 +40,17 @@ if (typeof Object.assign != 'function') {
 }
 /* eslint-enable */
 
+function debounce(func) {
```

Comment:
Um, should I add another argument something like `debounce(func, debounceDuration)` ?